### PR TITLE
Elgato.GameCapture4K60ProMK2Driver version 1.1.0.194.1

### PR DESCRIPTION
**Pull request by krystofkrticka**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26821)

### CODE REVIEW

File: manifests/e/Elgato/GameCapture4K60ProMK2Driver/1.1.0.194.1/Elgato.GameCapture4K60ProMK2Driver.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCapture4K60ProMK2Driver
```
Comment (OfficialEsco) [inline manifests/e/Elgato/GameCapture4K60ProMK2Driver/1.1.0.194.1/Elgato.GameCapture4K60ProMK2Driver.installer.yaml:4]:
```suggestion
PackageIdentifier: Elgato.GameCapture4K60ProMK2
```

File: manifests/e/Elgato/GameCapture4K60ProMK2Driver/1.1.0.194.1/Elgato.GameCapture4K60ProMK2Driver.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCapture4K60ProMK2Driver
```
Comment (OfficialEsco) [inline manifests/e/Elgato/GameCapture4K60ProMK2Driver/1.1.0.194.1/Elgato.GameCapture4K60ProMK2Driver.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Elgato.GameCapture4K60ProMK2
```

File: manifests/e/Elgato/GameCapture4K60ProMK2Driver/1.1.0.194.1/Elgato.GameCapture4K60ProMK2Driver.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCapture4K60ProMK2Driver
```
Comment (OfficialEsco) [inline manifests/e/Elgato/GameCapture4K60ProMK2Driver/1.1.0.194.1/Elgato.GameCapture4K60ProMK2Driver.yaml:4]:
```suggestion
PackageIdentifier: Elgato.GameCapture4K60ProMK2
```

File: manifests/e/Elgato/GameCapture4K60ProMK2Driver/1.1.0.194.1/Elgato.GameCapture4K60ProMK2Driver.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCapture4K60ProMK2Driver
+PackageVersion: 1.1.0.194.1
+PackageLocale: en-US
+Publisher: Elgato Systems
+PackageName: Elgato Game Capture 4K60 Pro MK.2 Driver Installer
```
Comment (OfficialEsco) [inline manifests/e/Elgato/GameCapture4K60ProMK2Driver/1.1.0.194.1/Elgato.GameCapture4K60ProMK2Driver.locale.en-US.yaml:8]: image
```suggestion
PackageName: Game Capture 4K60 Pro MK.2
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Remember to change the software folder name

File: manifests/e/Elgato/GameCapture4K60ProMK2/1.1.0.194.1/Elgato.GameCapture4K60ProMK2.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCapture4K60ProMK2
+PackageVersion: 1.1.0.194.1
```
Comment (ItzLevvie) [inline manifests/e/Elgato/GameCapture4K60ProMK2/1.1.0.194.1/Elgato.GameCapture4K60ProMK2.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.1.0.194
```

File: manifests/e/Elgato/GameCapture4K60ProMK2/1.1.0.194.1/Elgato.GameCapture4K60ProMK2.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCapture4K60ProMK2
+PackageVersion: 1.1.0.194.1
```
Comment (ItzLevvie) [inline manifests/e/Elgato/GameCapture4K60ProMK2/1.1.0.194.1/Elgato.GameCapture4K60ProMK2.yaml:5]:
```suggestion
PackageVersion: 1.1.0.194
```

File: manifests/e/Elgato/GameCapture4K60ProMK2/1.1.0.194.1/Elgato.GameCapture4K60ProMK2.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCapture4K60ProMK2
+PackageVersion: 1.1.0.194.1
```
Comment (ItzLevvie) [inline manifests/e/Elgato/GameCapture4K60ProMK2/1.1.0.194.1/Elgato.GameCapture4K60ProMK2.installer.yaml:5]:
```suggestion
PackageVersion: 1.1.0.194
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**